flow-typed/box-ui-elements.js

```diff
@@ -654,6 +654,7 @@ type TaskMini = {|
 type TaskCollabRole = 'CREATOR' | 'ASSIGNEE';
 
 type TaskCollab<R> = {|
+    completed_at?: ?ISODate,
```

Comment:
I'm likely missing the context of the data structure, but should this be on the task assignment object rather than the collaborator?

Reply:
The collaborator object in the new service has role="assignee|creator", it is the replacement for the old assignment table